lib/dark_worlds_server/runner_supervisor/runner.ex

```diff
@@ -228,47 +223,14 @@ defmodule DarkWorldsServer.RunnerSupervisor.Runner do
     {:stop, {:shutdown, :game_timeout}, state}
   end
 
-  def handle_info({:spawn_bots, bot_count}, state) when bot_count > 0 do
-    {:ok, bot_handler_pid} = BotPlayer.start_link(self(), %{})
-
-    {game_state, bots_ids} =
-      Enum.reduce(0..(bot_count - 1), {state.game_state, []}, fn _, {acc_game_state, bots} ->
-        # character = Enum.random(["h4ck", "muflus"])
-        character = "muflus"
-        {new_game_state, player_id} = GameBackend.add_player(acc_game_state, character)
-
-        {new_game_state, [player_id | bots]}
-      end)
-
-    Process.send_after(self(), {:activate_bots, bots_ids}, 10_000)
```

Comment:
Bots had this delay on the bots activation since the player won't be able to move for 10 seconds after we enter the match since we have the "Prepare for battle" screen + the camera cinematic, with this change we're activating the bots instantly, do you think we should fix this on this pr?